### PR TITLE
Compute the function yourParticipation

### PR DESCRIPTION
**Pull request by shtukas**
Notes:

1. The userStats objects's values are strings. So we need to convert them back to Integer for the addition. I will update the structure in a coming PR.

2. @mchv Pointed out that there might be a better way to get the first 3 digits of a decimal number. Currently the operation is abstcted in a separate function: `decimalTo3Digit`

### CODE REVIEW

File: ui/app/com/gu/recipeasy/views/app.scala.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
 @import com.gu.recipeasy.models._
 @(title: String, userFirstname:String, progressBarPercentage: Double, generalStats: Map[GeneralStatisticsPoint,Long], userStats: Map[PersonalStatisticsPoint, String])
 
+@decimalTo3Digit(number: Double) = @{
+    ((number)*1000).toInt.toDouble/1000
```
Comment (shtukas) [inline ui/app/com/gu/recipeasy/views/app.scala.html]: Awesome. Let me try!

Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/views/app.scala.html]: If not, there will be lots of Java classes that have this functionality such as `https://docs.oracle.com/javase/7/docs/api/java/text/NumberFormat.html`

**Comment by LATaylor-guardian** [comment]
Aside from my comment if it proves helpful 👍

**Comment by shtukas** [comment]
@LATaylor-guardian The c-style number formatting did work. Thanks!

**Pull request merged**